subsys/bluetooth/controller/ll_sw/ull_llcp_common.c

```diff
@@ -115,6 +119,43 @@ static void lp_comm_tx(struct ull_cp_conn *conn, struct proc_ctx *ctx)
 	tx_enqueue(conn, tx);
 }
 
+static void lp_comm_ntf_feature_exchange(struct ull_cp_conn *conn, struct proc_ctx *ctx, struct pdu_data *pdu)
+{
+	switch (ctx->response_opcode) {
+	case PDU_DATA_LLCTRL_TYPE_FEATURE_RSP:
+		ntf_encode_feature_rsp(conn, pdu);
+		break;
+	case PDU_DATA_LLCTRL_TYPE_SLAVE_FEATURE_REQ:
+	case PDU_DATA_LLCTRL_TYPE_FEATURE_REQ:
+		ntf_encode_feature_req(conn, pdu);
+		break;
+	case PDU_DATA_LLCTRL_TYPE_UNKNOWN_RSP:
+		ntf_encode_unknown_rsp(ctx, pdu);
+		break;
+	default:
+		/* EGON: TODO: define behaviour for unexpected PDU */
+		LL_ASSERT(0);
+	}
+}
+
+static void lp_comm_ntf_version_ind(struct ull_cp_conn *conn,  struct proc_ctx *ctx, struct pdu_data *pdu)
+{
+	switch (ctx->response_opcode) {
+	case PDU_DATA_LLCTRL_TYPE_VERSION_IND:
+		ntf_encode_version_ind(conn, pdu);
+		break;
+	default:
+		/* EGON: TODO: define behaviour for unexpected PDU */
+		LL_ASSERT(0);
+	}
+}
+
+static void lp_comm_ntf_unknown_proc(struct ull_cp_conn *conn,  struct proc_ctx *ctx, struct pdu_data *pdu)
```

Comment:
Removed the complete procedure since it is not used; I had hoped to implement Unknown Response, but that requires some more work, in particular for testing it